Source/itemdat.cpp

```diff
@@ -38,6 +38,9 @@ tl::expected<item_drop_rate, std::string> ParseItemDropRate(std::string_view val
 	if (value == "Never") return IDROP_NEVER;
 	if (value == "Regular") return IDROP_REGULAR;
 	if (value == "Double") return IDROP_DOUBLE;

```

Comment:
How about we just replace this with an int, 0, 1, 2? The TSV support isn't in a release version yet so the format isn't considered stable yet.

Reply:
adds a bit to the diff but lets us remove a function in the parsing code, so net improvement imo :)